.eslintrc.js

```diff
@@ -1,4 +1,8 @@
 module.exports = {
     root: true,
+    parser: '@babel/eslint-parser',
+    parserOptions: {
+        requireConfigFile: false
+    },
```

Comment:
`@babel/eslint-parser` is needed for object spread syntax, otherwise `eslint` throws parsing error for that kind of construct. 

Indeed, lets enable it for other repos updating the shared eslint config